README.md

```diff
@@ -0,0 +1,71 @@
+# Apache Beam (incubating) website
+
+This is the website for [Apache Beam](http://beam.incubator.apache.org)
+(incubating).
+
+### About this site
+The Beam website is built using [Jekyll](http://jekyllrb.com/). Additionally,
+for additional formatting capabilities, this website uses
+[Twitter Bootstrap](http://getbootstrap.com/).
+
+This website is presently hosted at:
+
+    http://beam.incubator.apache.org
+
+It is important to note there are two sets of "website code" when dealing with
+the Apache Beam website.
+
+1. The **Jekyll code** which contains all of the resources for building,
+testing, styling, and tinkering with the website. Think of it as a website SDK.
+1. The **static website** content which contains the raw content for the
+website - all compiled, structured, and ready to go. This is the content which
```

Comment:
"This is the content which is pushed for hosting as a static website". 

Is this accurate? Pushed where? This is probably unnecessary detail since it is not a manual action. Perhaps rephrase to something along the lines of:

"This is the content which serves the static website.", OR
"This is the static content is what is actually hosted on the Apache Beam website."